pkg/service/service.go

```diff
@@ -187,13 +188,97 @@ func (s *Service) GetCurrentTs() time.Time {
 	return time.Now()
 }
 
+func (s *Service) populateBackendMapV1FromV2(ipv4, ipv6 bool) error {
+	var (
+		count int
+		err   error
+	)
+
+	if ipv4 {
```

Comment:
I think we can avoid duplicating the code. A crude way:
```
foo := map[string]bool{"4": ipv4, "6":ipv6}
for k, v := range foo {
if !v {
  continue
}

var map *bpf.Map
var kkey lbmap.BackendKey

if v == "4" {
  map = lbmap.Bakckend4Map
  kkey = key.(*lbmap.Backend4KeyV2)
} else if v == "6" {
 ...
}

rest of the code

}

Reply:
Done, PTAL :)